index.html

```diff
@@ -2880,9 +2893,18 @@
           further action. The <a>user agent</a> user interface should ensure
           that this never occurs.
           </li>
-          <li>
-            <a>Fire an event</a> named <var>name</var> at <var>request</var>
-            using <a>PaymentRequestUpdateEvent</a>.
+          <li>Let <var>event</var> be the result of <a>fire an event</a> named
+          <var>name</var> at <var>request</var> using
+          <a>PaymentRequestUpdateEvent</a>.
+          </li>
+          <li data-link-for="PaymentRequestUpdateEvent">If
+          <var>event</var>.<a>[[\waitForUpdate]]</a> is true, disable any part
+          of the user interface that could cause another update event to be
+          fired. Wait for <var>event</var>.<a>updateWith()</a> method's
+          <var>detailsPromise</var> to settle.
```

Comment:
", then re-enable the relevant parts of user interface", I think.

Maybe this should be a series of nested steps under the "If ... is true".

Reply:
`.updateWith()` controls re-enabling the user interface, in step 14. It might be redundant to re-state it here.

Reply:
Ah ok, in that case, agreed.